### PR TITLE
Change naming pattern for continuation fields and do not generate them for unused parameters

### PR DESCRIPTION
**Pull request by CommanderTvis**

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -1,5 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="ASMIdeaPluginConfiguration">
```
Comment (ilmirus) [inline .idea/misc.xml:3]: Please, do not include irrelevant changes.

File: compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/lower/SuspendLambdaLowering.kt [hunk 2]
```diff
@@ -144,6 +144,7 @@ private class SuspendLambdaLowering(context: JvmBackendContext) : SuspendLowerin
             }
         }
 
+    @OptIn(ObsoleteDescriptorBasedAPI::class)
```
Comment (ilmirus) [inline compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/lower/SuspendLambdaLowering.kt:147]: Looks like this can be removed

**Review by ilmirus — CHANGES_REQUESTED** [review]

File: .idea/misc.xml (hunk 1 above)
Comment (CommanderTvis) [reply .idea/misc.xml:3]: Already discarded ^^^

File: compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/lower/SuspendLambdaLowering.kt (hunk 2 above)
Comment (CommanderTvis) [reply compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/lower/SuspendLambdaLowering.kt:147]: OK

**Review by ilmirus — APPROVED** [review]

**Comment by ilmirus** [comment]
Thanks! Manually rebased and pushed to master.

**Pull request closed without merging**